### DISCUSSION TITLE
Welcome to Oraxen Discussions!

### DISCUSSION THREAD

**Discussion by Th0rgal** [oraxen/oraxen#92] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Th0rgal** [comment]
My name is Thomas, I'm the original developer of the Oraxen plugin. :)

**Reply by fichdlmaa** [reply]
Hello Thomas / Th0rgal / Uxon,

mny tnx for Oraxen. 

_I'm german IT professional and MC user since about java edition 1.7.9 and run small Spigot servers since Spigot 1.8.8
Spigot 1.15.2 is running on a small private server (see https://www.netzplatz.net) from time to time reachable via internet ..._

I'm now preparing for 1.16.5 (test installation on my Debian Buster workstation) and wanted to test oraxen with it, but: 
The needed ressource pack won't load - nor automatically neither via proposed command. (latest 1.68.0)

Perhaps I'm making some mistake or the ressi pack link is broken?
Any "special" setting(s) or extraordinary ports for communication with server?
There is *no* error message except the unsuccesful info after failed download

Greets

MC/Spigot/Github/Polymart/... FichdlMaa

**Reply by Th0rgal** [reply]
> Hello Thomas / Th0rgal / Uxon,
> 
> mny tnx for Oraxen.
> 
> _I'm german IT professional and MC user since about java edition 1.7.9 and run small Spigot servers since Spigot 1.8.8 Spigot 1.15.2 is running on a small private server (see https://www.netzplatz.net) from time to time reachable via internet ..._
> 
> I'm now preparing for 1.16.5 (test installation on my Debian Buster workstation) and wanted to test oraxen with it, but:
> The needed ressource pack won't load - nor automatically neither via proposed command. (latest 1.68.0)
> 
> Perhaps I'm making some mistake or the ressi pack link is broken?
> Any "special" setting(s) or extraordinary ports for communication with server?
> There is _no_ error message except the unsuccesful info after failed download
> 
> Greets
> 
> MC/Spigot/Github/Polymart/... FichdlMaa

Hello sir,

Did you buy the plugin or build it by yourself? The source code doesn't include a resourcepack because textures and models were bought using money made from the sales This might be the issue.

Sincerely,
Thomas

**Reply by fichdlmaa** [reply]
Allô Thomas ;-)

je l'ai acheté via spigotmc.org -> https://www.spigotmc.org/members/fichdlmaa.587740/

OK. La semaine prochaine, je vais effaçer complètement le fichier "plugins/oraxen/" et veux voire si ça va après restart...

À bientôt donc!

Le FichdlMaa

**Reply by fichdlmaa** [reply]
Bonjour Thomas,

maintenant j'ai trouvé la raison: Oraxen et Quality Armory (QA) sont incompatibles l'un avec l'autre :-(


Avec tous les deux, Oraxen ne réussi pas à intégrer son Ressi-Pack. QA charge son Ressi-Pack, mais le plugin ne fonctionne pas correctement. -- Oraxen va bien seul, QA aussi :triste:

Quel dommage. Si Oraxen offrait quelques armes à feu, je n'aurais pas besoin le QA ...

-> https://www.spigotmc.org/resources/quality-armory.47561 

Ma seul possibilité est de t'implorer si tu veux contacter zombie_striker à ce problème.
(Qui - à part moi - a besoin de 2 plugins d'armes)

-> https://www.spigotmc.org/resources/authors/zombie_striker.62300/


_Mes salutations de la belle Franconie 

Le FichdlMaa_

**Reply by Th0rgal** [reply]
> Bonjour Thomas,
> 
> maintenant j'ai trouvé la raison: Oraxen et Quality Armory (QA) sont incompatibles l'un avec l'autre :-(
> 
> Avec tous les deux, Oraxen ne réussi pas à intégrer son Ressi-Pack. QA charge son Ressi-Pack, mais le plugin ne fonctionne pas correctement. -- Oraxen va bien seul, QA aussi :triste:
> 
> Quel dommage. Si Oraxen offrait quelques armes à feu, je n'aurais pas besoin le QA ...
> 
> -> https://www.spigotmc.org/resources/quality-armory.47561
> 
> Ma seul possibilité est de t'implorer si tu veux contacter zombie_striker à ce problème.
> (Qui - à part moi - a besoin de 2 plugins d'armes)
> 
> -> https://www.spigotmc.org/resources/authors/zombie_striker.62300/
> 
> _Mes salutations de la belle Franconie
> 
> Le FichdlMaa_

Je peux rajouter des armes :)

**Comment by fichdlmaa** [comment]
Ah. Trés bien :-D

pas necessaire d'être si "compliqué / large" comme dans QA:

p. ex.
- même calibre (munition) pour tous
- un fusil assaut et sniper chacun, 
- un pistolet ordinair et à tir rapide (comme uzi) chacun
- tous les 4 sont armes imaginaires (modèles et skins), pas armes existent en réalitée.

Oui, ce serait superbe, quelque chose comme ça déja inclus dans Oraxen 1.8 :-D 


À bientôt donc!

Le FichdlMaa

**Comment by snwsue** [comment]
Hey I’m not sure why but apparently I’m banned from the discord my discord name is snwsue and I’ve bought the plugin oraxen please can you help me?

**Reply by Boy0000** [reply]
no clue when or why u were banned but its been revoked

**Comment by JeffersonSerrano** [comment]
Hi! Is there a way for me to get the CustomBlockExpansion mentioned in the docs?
